build/deploy.sh

```diff
@@ -14,7 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set -o errexit -o nounset -o pipefail
+set -o errexit -o nounset -o pipefail -o xtrace
```

Comment:
Do we want the `xtrace` here all the time? I assume that's just left over debugging.  If we really want something like that, it'd probably be:

```suggestion
set -o errexit -o nounset -o pipefail ${XTRACE:+-o xtrace}
```

Reply:
Yeah, leftover. I'll take the suggestion, though.